_posts/2021-4-19-Actors-in-the-MCU.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+author: Amna Amin
+excerpt_separator: <!--more-->
+title: Actors in the MCU 
+---
+
+# Research Summary
+
+The research question I wanted to investigate was which actors in the Marvel Cinemetic Universe (MCU) were in which films. 
+In particular, I wanted to see if certain actors were in more films and which films had the biggest ensemble casts. 
+For instance, I expected the Avengers movies to have a lot of actors, and I expected those actors to have also been in other MCU movies. I wanted to investigate that
+question since the MCU is one of the biggest and impactful franchises in the modern day. Thus, the actors cast in the movies, especially the ones who appear most frequently, are influential celebrities.
+Additionally, I thought it was interesting to see actors and the movies they were cast in during the silent era, and examining actors in the MCU seemed like the modern day equivalent. 
+
+The most interesting nodes to me were the avenger film nodes since those movies had the largest casts since they were bigger ensemble films. 
+The most interesting edges were between the most popular actors like Robert Downey, Mark Ruffalo, and Scarlett Johansson, especially since these actors sometimes appeared in movies that were not avengers movies or movies where they were the star. 
+For instance, Robert Downey had an interesting edge between himself and the Spiderman movie. Similarly, Mark Ruffalo had an edge between himself and Thor Ragnarok. 
+These nodes are signficant since they showcase two important aspects of the narrative of the research. The first is that it shows how certain actors and characters are popular so the MCU chooses to involve them in more projects. 
+The second is that certain characters are important instigators and features of multiple stories. For example, Robert Downey as Iron Man is important for helping Spider Man become a hero. 
+
+I was a bit surprised to see the amount of female characters cast in multiple films. For instance, I was surprised to see that Gwenyth Paltrow was in so many movies since I did not expect a hero's love interest to be important enough to cast in a lot of films.
+I was also surprised to see that Scarlett Johansson was cast in as many films as she was especially since some of them were not Avengers ensemble movies. For instance, she was in one of the Iron Man movies. 
+I think this aspect could demonstrate that the MCU wants to include more female characters, but they use the same few in multiple films. 
+
+# Network Image 
+![Network Image](https://github.com/amnaa329/HUM-331-Princeton.github.io/blob/67ffdaff0c4d19b8b38f68639909ae57947d39eb/images/Network%20Image.png) 
```

Comment:
You're going to need to change this URL from the URL for your GitHub repository file to the URL for hte image itself (https://github.com/amnaa329/HUM-331-Princeton.github.io/blob/67ffdaff0c4d19b8b38f68639909ae57947d39eb/images/Network%20Image.png?raw=true).